src/Renci.SshNet.Tests/Classes/ClientAuthenticationTest.cs

```diff
@@ -28,7 +28,7 @@ public void Ctor_PartialSuccessLimit_Zero()
             catch (ArgumentOutOfRangeException ex)
             {
                 Assert.IsNull(ex.InnerException);
-                Assert.AreEqual(string.Format("Cannot be less than one.{0}Parameter name: {1}", Environment.NewLine, ex.ParamName), ex.Message);
+                Assert.AreEqual(new ArgumentOutOfRangeException("partialSuccessLimit", "Cannot be less than one.").Message, ex.Message);
```

Comment:
I propose to add an **ArgumentExceptionAssert** class with a static `MessageEquals(string expected, ArgumentException )` that - under the hood - creates a new **ArgumentException** with the expected message and the **ParamName** of the specified **ArgumentException** and asserts whether the **Message** of that newly created **ArgumentException** matches the **Message** of the specified **ArgumentException**.